.pre-commit-config.yaml

```diff
@@ -41,10 +41,12 @@ repos:
         language: system
         types: [python]
 
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.13.2
+  - repo: local
     hooks:
       - id: isort
+        name: isort
+        entry: bash -c "poetry run isort ."
+        language: system
```

Comment:
If you rebase, this would catch issues from the fern generated files, presumably bc it missed our config